apex-app-archetype/pom.xml

```diff
@@ -50,5 +50,22 @@
         </plugin>
       </plugins>
     </pluginManagement>
+
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+        <includes>
+          <include>META-INF/maven/archetype-metadata.xml</include>
```

Comment:
will it be better to separate the filtered resource into a separate directory like filtered-resources, so it is more explicit what resources are filtered?